package.json

```diff
@@ -28,5 +28,8 @@
   },
   "engines": {
     "node": ">=18"
+  },
+  "dependencies": {
+    "zod": "^3.24.1"
```

Comment:
Is this needed here? And we can use the catalog version?

Reply:
Nice catch! Indeed an oversight, thanks